### PR TITLE
Add changelog entry for cron tasks endpoints

### PR DESCRIPTION
**Pull request by TheoEb**
Merge the dev doc at the same time as this PR: https://github.com/Scalingo/developers-documentation/pull/162

### CODE REVIEW

File: changelog/api/_posts/2021-06-28-cron-tasks.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
```
Comment (EtienneM) [inline changelog/api/_posts/2021-06-28-cron-tasks.md:1]: Title of the file is wrong

**Review by EtienneM — COMMENTED** [review]
Let's wait for the feature to be released before merging this one

**Comment by Soulou** [comment]
Will be merged only when the feature is in prod

**Comment by Soulou** [comment]
Depends of the answers for https://github.com/Scalingo/developers-documentation/pull/162

**Review by EtienneM — APPROVED** [review]

**Pull request merged**